src/test/scala/org/camunda/feel/impl/interpreter/InterpreterExpressionTest.scala

```diff
@@ -60,6 +60,12 @@ class InterpreterExpressionTest
     eval("x instance of string", Map("x" -> 0)) should be(ValBoolean(false))
   }
 
+  it should "be a instance of Any should always pass" in {
+    eval("x instance of Any", Map("x" -> "yes")) should be(ValBoolean(true))
+    eval("x instance of Any", Map("x" -> 1)) should be(ValBoolean(true))
+    eval("x instance of Any", Map("x" -> true)) should be(ValBoolean(true))
+  }
```

Comment:
:+1: 

Please add one more case for `null instance of Any`.

Reply:
Added extra test and fixed check for `null`.